runtime/near-vm-runner/src/tests.rs

```diff
@@ -119,23 +119,26 @@ fn make_simple_contract_call_vm(
     code: &[u8],
     method_name: &str,
     vm_kind: VMKind,
-) -> (Option<VMOutcome>, Option<VMError>) {
+) -> (VMOutcome, Option<VMError>) {
```

Comment:
I am a bit worried that now we return an outcome even for catastrophic errors, but I think this is a fine intermediate point!

Reply:
Agreed on that, feels wrong... A good motivation to keep going with the refactor!